include/veriblock/entities/coin.hpp

```diff
@@ -16,6 +16,10 @@ struct Coin {
     return Coin(readSingleBEValue<int64_t>(stream));
   }
 
+  static void toVbkEncoding(WriteStream& stream, const Coin& value) {
```

Comment:
do not make it static, and you will not have to provide `Coin` instance. 
Use `this` instead of `Coin& value` argument